crossbar/webservice/base.py

```diff
@@ -62,11 +62,8 @@ class ResourceFallback(File):
     """
     Handle requests for non-existent URL's
     """
-    def __init__(self, path, config, **kwargs):
+    def __init__(self, path, **kwargs):
```

Comment:
we can completely remove `ResourceFallback` (see also the other comment below)

Reply:
done